extra/exports

```diff
@@ -12,6 +12,7 @@
 base64_bufsize
 base64_decode
 base64_encode
+box_dd_version_id
```

Comment:
You write in the commit message that
```
There's a special case for upgrade() during bootstrap() - triggers are
disabled during bootstrap, that's why dd_version_id is not being updated.
Handle this by passing _initial_version=1.7.5 to the upgrade function.
```

Does it mean that after `box.internal.bootstrap()` when we're trying to save a new bootstrap.snap, for example with schema version 2.12.0, `box_dd_version_id` will stay 2.11.0?

This doesn't matter much, I'm just trying to understand.

Reply:
No, if we add a new entry to `handlers` (e.g. `upgrade_to_2_12_0`) and launch Tarantool at clean directory, then during `box.cfg{}` the new handler `upgrade_to_2_12_0` will be called and `box_dd_version_id` will become 2.12.0.

Reply:
So `box_dd_version_id` will be 2.12.0 both before and after `box.internal.bootstrap()`?

Reply:
Yes

Reply:
Ok, thanks for the explanation!